test/integration/fapi-get-esys-blobs.int.c

```diff
@@ -43,14 +43,19 @@ get_json_hex_string(const uint8_t *buffer, size_t size)
 
 static TSS2_RC
 auth_callback(
-    FAPI_CONTEXT *context,
+    char const *objectPath,
     char const *description,
-    char **auth,
+    const char **auth,
     void *userData)
 {
     (void)description;
     (void)userData;
-    *auth = strdup(PASSWORD);
+
+    if (!objectPath) {
+        return_error(TSS2_FAPI_RC_BAD_VALUE, "No path.");
+    }
+
+    *auth = PASSWORD;
     return_if_null(*auth, "Out of memory.", TSS2_FAPI_RC_MEMORY);
```

Comment:
So in this case we can remove all these return_if_null(*auth, "Out of memory.", ...) here and below.
These are no longer needed.